pom.xml

```diff
@@ -992,15 +1026,45 @@
                 <hbase.version>1.2.0-cdh5.12.0</hbase.version>
                 <hive.version>1.1.0-cdh5.12.0</hive.version>
                 <zookeeper.version>3.4.5-cdh5.12.0</zookeeper.version>
+                <spark.major.version>spark2.1</spark.major.version>
                 <spark.version>2.1.0.cloudera1</spark.version>
                 <kafka.version>0.9.0-kafka-2.0.2</kafka.version>
             </properties>
             <modules>
                 <module>hbase_storage</module>
                 <module>hbase_pipeline</module>
                 <module>hbase_sql</module>
+                <module>splice_spark</module>
             </modules>
         </profile>
+        <profile>
+            <id>cdh5.13.0</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>cdh5.13.0</value>
+                </property>
+            </activation>
+            <properties>
+                <cdh.parcel.version>5.13.0</cdh.parcel.version>
+                <envClassifier>cdh5.13.0</envClassifier>
+                <envHbase>hbase1.2.0</envHbase>
+                <hadoop.version>2.6.0-cdh5.13.0</hadoop.version>
+                <hbase.version>1.2.0-cdh5.13.0</hbase.version>
+                <hive.version>1.1.0-cdh5.13.0</hive.version>
+                <zookeeper.version>3.4.5-cdh5.13.0</zookeeper.version>
+                <spark.major.version>spark2.1</spark.major.version>
+                <spark.version>2.1.0.cloudera1</spark.version>
```

Comment:
i thought we wanted to move forward to spark 2.2.0.cloudera2. discuss?

Reply:
We do, I will fix.